src/main/java/easy/mind/com/api/DTO/conversion/UserDtoToUser.java

```diff
@@ -0,0 +1,29 @@
+package easy.mind.com.api.DTO.conversion;
+
+import easy.mind.com.api.DTO.UserDTO;
+import easy.mind.com.api.Entity.User;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public final class UserDtoToUser {
+
+    public static User convertToUser(UserDTO userDTO) {
+        return User.builder()
+                .id(userDTO.getId())
+                .firstName(userDTO.getFirstName())
+                .lastName(userDTO.getLastName())
+                .email(userDTO.getEmail())
+                .password(userDTO.getPassword())
+                .build();
+    }
+
+    public static UserDTO convertToUserDTO(User user) {
+        return UserDTO.builder()
+                .id(user.getId())
+                .firstName(user.getFirstName())
+                .lastName(user.getLastName())
+                .email(user.getEmail())
+                .password(user.getPassword())
```

Comment:
I'm not sure that exposing users' passwords is a good idea. 
You can just place here some placeholder "**********"